ext/gd/gdcache.h

```diff
@@ -41,6 +41,7 @@
 /*********************************************************/
 
 #if (!defined(_OSD_POSIX) && !defined(__FreeBSD__)) && HAVE_MALLOC_H
+#elif !defined(__OpenBSD__)
```

Comment:
This doesn't look right - won't this break the condition for FreeBSD etc? Shouldn't the OpenBSD check be integrated in the previous line instead?